app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>All Contestants</h1>
+
+<% @contestants.each do |contestant| %>
+  <p><%= contestant.name %></p><br>
```

Comment:
This is almost there for US 2 but needs the project info. You've set up your many to many relationship correctly so you should easily be able to access a contestant's projects using `contestant.projects` and iterate through them to display.